### PR TITLE
Updated parquet version and fixed dependencies to use new features li…

### PR DESCRIPTION
**Pull request by ajitgogul**
…ke rowcount and size

### CODE REVIEW

**Comment by ajitgogul** [comment]
Mainly aimed to make use of the rowcount and size feature introduced in parquet 1.10.0 : https://issues.apache.org/jira/browse/PARQUET-196

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: scala
 scala:
   - 2.11.12
-jdk: oraclejdk7
+jdk: oraclejdk9
```
Comment (regadas) [inline .travis.yml]: For this part can you add a matrix?

File: build.sbt [hunk 2]
```diff
@@ -2,10 +2,10 @@ organization := "com.spotify.data"
 name := "gcs-tools"
 version := "0.1.8-SNAPSHOT"
 
-val gcsVersion = "1.6.2-hadoop2"
+val gcsVersion = "1.6.3-hadoop2"
 val hadoopVersion = "2.7.4"
-val avroVersion = "1.8.2"
-val parquetVersion = "1.9.0"
+val avroVersion = "1.9.0"
+val parquetVersion = "1.10.1"
```
Comment (regadas) [inline build.sbt]: we can even possibly bump this to 1.11.0

File: build.sbt [hunk 3]
```diff
@@ -2,10 +2,10 @@ organization := "com.spotify.data"
 name := "gcs-tools"
 version := "0.1.8-SNAPSHOT"
 
-val gcsVersion = "1.6.2-hadoop2"
+val gcsVersion = "1.6.3-hadoop2"
 val hadoopVersion = "2.7.4"
-val avroVersion = "1.8.2"
-val parquetVersion = "1.9.0"
+val avroVersion = "1.9.0"
```
Comment (regadas) [inline build.sbt]: let's not bump avro in this PR, parquet up to 1.11.0 uses 1.8.2

avro 1.9 as some breaking changes

File: build.sbt (hunk 2 above)
Comment (ajitgogul) [reply build.sbt]: sure.

File: build.sbt (hunk 3 above)
Comment (ajitgogul) [reply build.sbt]: sure. thanks for the review. will update.

File: build.sbt (hunk 2 above)
Comment (ajitgogul) [reply build.sbt]: for some reason, sbt is not able to find 1.11.0

**Comment by ajitgogul** [comment]
- Bumped parquet version. Avro & Protobuf remain unchanged.
- Travis build was failing for oraclejdk7. So updated to matrix with oraclejdk9.
- sbt build version < 1.x had issues with oraclejdk9. updated build version.
- Updated scala version in build and travis.yml
- Updated README with the correct versions for parquet, avro and protobuf
- Fixed build.sbt syntax as per sbt 1.x as the older one was no longer supported

File: .travis.yml (hunk 1 above)
Comment (ajitgogul) [reply .travis.yml]: fixed.

**Pull request merged**